lms/djangoapps/mobile_api/tests.py

```diff
@@ -3,49 +3,9 @@
 """
 
 import ddt
-from mock import patch
 from django.test import TestCase
 
-from xmodule.modulestore.tests.factories import CourseFactory
-
-from .utils import mobile_access_when_enrolled, mobile_course_access, mobile_view
-from .testutils import MobileAPITestCase, ROLE_CASES
-
-
-@ddt.ddt
-class TestMobileAccessWhenEnrolled(MobileAPITestCase):
-    """
-    Tests for mobile_access_when_enrolled utility function.
-    """
-    @ddt.data(*ROLE_CASES)
-    @ddt.unpack
-    def test_mobile_role_access(self, role, should_have_access):
```

Comment:
Should these tests be updated to apply to is_mobile_available_for_user instead of just deleted?